frontend/src/store/creator/reducer.js

```diff
@@ -59,7 +60,9 @@ export function reducer(state, action) {
 			...(currentStep[prefixedProperty] ||
 				Array(state.model.file_frames[0].edges_vertices.length).fill(null)),
 		]
-		values[state.selectedEdge] = value
+		for (let selectedEdge of edges) {
```

Comment:
Just being tedious here, I wouldn't call this `selectedEdge` since nothing in this function implies that you carry out modifications only on the selected edges (it's more generic).